### PR TITLE
Add missing Esperanto strings

### PR DESCRIPTION
**Pull request by yellowcrash10**
I have translated some strings that were missing from the Esperanto locale.

### CODE REVIEW

File: eo.json [hunk 1]
```diff
@@ -1338,6 +1343,9 @@
         "Tutorial.AvatarTutorial.EquippingAndFavoriting.Description" : "Post kiam vi havos avataron, ekportu ĝin klakante sur ĝi kaj elektante Porti Avataron.<br><br>Konservu vian nun portatan avataron alklakante <sprite name=Download>  .<br><br>Vi povas preferatigi vian avataron elektante ĝin en via<br>inventaro kaj alklakante <sprite name=Favorite>  .",
         "Tutorial.AvatarTutorial.AvatarCreation.Description" : "Por importi kaj agordi propran avataron, iru al<br>la Lernilo pri Kreado de Neos-avataroj",
 
+        "Tutorial.Intro.PublicFolders.Title" : "Publikaj Dosierujoj",
+        "Tutorial.Intro.PublicFolders.Description" : "Publikaj Dosieruoj estas facila vojo por uzantoj en la Komunumo konigi kaj ĝisdatigi iliajn Faraĵojn kiel Avataroj.<br><br>Por konservi, ekprenu la dosierujon, kaj dum tenado premu <sprite name=Add>  en via Inventaro.<br><br>Vi ankaŭ povas konservi per via Kunteksta Menuo.<br><br>Provu per <b>The Starter Pack</b>. Dosierujo desegnita por novaj uzantoj.",
```
Comment (Inferus25) [inline eo.json:1347]: The last phrase should be "Dosierujo _dezajnita_ [...]" instead of "desegnita"

**Review by Inferus25 — CHANGES_REQUESTED** [review]
Change "desegnita" with "dezajnita"

**Comment by yellowcrash10** [comment]
Dankon! I have made the correction.

File: eo.json (hunk 1 above)
Comment (yellowcrash10) [reply eo.json]: Dankon! I have made the correction.

**Comment by Frooxius** [comment]
Thank you for the update!

**Pull request merged**